### DISCUSSION TITLE
Flash firmware

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1252] [Q&A]
Hi, I want to flash the offical firmware 2024.2 but why the crazyflie will appear the video situation ? It will disconnect and restart but nothing chang. 

https://github.com/bitcraze/discussions/assets/118436188/87bf0be5-1bfb-4edc-9449-c0aa153ac29e

**Comment by OliLov** [comment]
Same here, @CalvinKoung. I use the previous firmware for now. I tried to flash with the AI deck attached, but that doesn't seem to be the problem. But keep me updated if there is a workaround.

**Reply by CalvinKoung** [reply]
Hi, I later successfully refreshed it manually.
[[]](https://www.bitcraze.io/documentation/tutorials/getting-started-with-development/)(url)

**Comment by gemenerik** [comment]
Can you copy the terminal output here? Are you running the latest client and lib?

**Reply by CalvinKoung** [reply]
I use the version 2022.5 this VM. Because it system more stable.

**Reply by knmcguire** [reply]
@CalvinKoung if you want to flash the latest official firmware, you really need to update the cfclient to the latest as well. There is an important update to the NRF bootloader and older versions of the cfclient are not able to flash that. 

Please check out this blogpost https://www.bitcraze.io/2024/02/2024-2-firmware-release-with-new-bluetooth-stack/
